apollo-router/src/query_planner/fetch.rs

```diff
@@ -100,7 +100,18 @@ pub(crate) type SubgraphSchemas = HashMap<String, SubgraphSchema>;
 
 pub(crate) struct SubgraphSchema {
     pub(crate) schema: Arc<Valid<apollo_compiler::Schema>>,
-    pub(crate) implementers_map: HashMap<Name, apollo_compiler::schema::Implementers>,
+    // TODO: Ideally should have separate nominal type for subgraph's schema hash
+    pub(crate) hash: Arc<SchemaHash>,
```

Comment:
The `SchemaHash` struct already contains an `Arc` field, so wrapping it in another `Arc` is redundant

Reply:
good catch! will fix

Reply:
Oh sorry @IvanGoncharov I already pushed it! 🙈